include/State.h

```diff
@@ -32,7 +30,7 @@ class State {
     ul gridSize;
 
 private:
-    grid<uint8_t> _W; // 1 - cell alive
+    grid<bool> _W; // 1 - cell alive
```

Comment:
Skąd ta zmiana?

Reply:
Tak przyjmuje wartości 0 i 1. Inaczej się wypisuje jak jest uint8 i bool? 
Z tym vectorem booli to jest taki problem, że oprócz tego, że ma inny interfejs, to jest toż odrobinę wolniejszy.

Reply:
Na razie zostaw. Jak zrobię benchmark, że jest istotnie wolniejsze, to zmienię